stylesheets/scss/_ting.scss

```diff
@@ -5,24 +5,40 @@
   }
 }
 
+.pane-ting-collection {
+  .ting-collection-inner-wrapper {
+    .view-mode-collection-list {
+      &:last-child {
+        padding-bottom: 20px;
```

Comment:
I fear this change is going down the wrong path.

The current production site has the right positioning of buttons. It seems like we have changed this with the latest changes and are now trying to patch things up instead of keeping them as they were.

![harry potter og dodsregalierne kobenhavns biblioteker 2016-06-10 11-37-21](https://cloud.githubusercontent.com/assets/73966/15960448/2b27f1d6-2f00-11e6-9905-2c190325e6e0.png)